dogfood/main.tf

```diff
@@ -86,7 +86,7 @@ resource "coder_agent" "dev" {
   arch = "amd64"
   os   = "linux"
 
-  dir = data.coder_parameter.repo_dir.value
+  dir = "/home/coder"
```

Comment:
When the directory is set to `/home/coder/coder`,
`startup_script` installs everything in the specified directory. As `/home/coder/coder` is a git repo, git complains about unstaged files, and we either have to delete those files or populate them in .gitignore.

Reply:
To get the same behavior I added `cd ${data.coder_parameter.repo_dir.value}` as the last line in `startup_script`

Reply:
found a workaround for this so that we do not have to change this.